docs/src/content/docs/guides/authoring-content.md

```diff
@@ -202,9 +202,140 @@ var fun = function lang(l) {
 ```
 ````
 
-```md
-Long, single-line code blocks should not wrap. They should horizontally scroll if they are too long. This line should be long enough to demonstrate this.
-```
+### Expressive Code features
+
+Starlight uses [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) to extend formatting possibilities for code blocks.
+Expressive Code’s text markers and window frames plugins are enabled by default.
+Code block rendering can be configured using Starlight’s [`expressiveCode` configuration option](/reference/configuration/#expressivecode).
+
+#### Text markers
+
+You can highlight specific lines or parts of your code blocks using text markers.
+There are three styles of marker: a default neutral colored highlight, a green highlight indicating code that was inserted, and a red highlight indicating code that was deleted.
+
+- [Mark entire lines & line ranges](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#marking-entire-lines--line-ranges):
+
+  ```js {2-3}
```

Comment:
I have a medium-strong preference for swapping all these examples, and showing the code you need to write first, and *then* what it looks like.

Feels more in keeping with helping people write something: see the code, THEN see the results it produces.

Reply:
I also do! The only issue is that the entire page currently uses this ordering so I felt unsure about changing this without also revisiting the whole page, which felt like something that should maybe be done in a dedicated PR?

Reply:
I agree with both of you! I also see value in swapping the examples, but we should do it in a separate PR after getting this one over the finish line. :)